week2/Homework/Json/j_index.js

```diff
@@ -0,0 +1,109 @@
+//console.log("Hello World!" , process.argv);
+fs = require('fs');
+let command = "";
+
+if (process.argv.length < 3)
+	command = "help" ;
+else
+	command = process.argv[2];
+
+function readFile(file_name){return new Promise(
+    (resolve, reject) => {
+		fs.readFile(file_name, 'utf8', function (err,data) {
+			  if (err) {
+				reject(err);
+			  }
+			  else{
+				  resolve(data);
+			  }
+		});
+    });
+}
+function writeFile(file_name , file_data){return new Promise(
+    (resolve, reject) => {
+		fs.writeFile(file_name, file_data, function (err) {
+			  if (err) {
+				reject(err);
+			  }
+			  else{

```

Comment:
Be careful about your formatting. It might seem like it doesn't matter (the computer doesn't care, after all), but code is meant to be read by others 😄 

If you're going to write `if ( condition ) {`, make sure you write ` } else {`. ie. if you put spaces in between the keywords and the brackets, always do it.

This is also the case on line 35, where you're written `switch(command){` (the parentheses kind of make it look like a function call, which it's not). I would rather see it written as `switch (command) {`, or even `switch ( command ) {`.

You've also used tabs for indentation, which is no big deal, but take a look at how it appears on github - it seems to be showing 8 spaces per tab, which is a lot, and breaks up your code quite a bit.